setup/installation.md

```diff
@@ -24,14 +41,28 @@ data, including environment information and session tokens.
    helm repo add coder https://helm.coder.com
    ```
 
-2. Install the helm chart into the cluster
+2. Install the helm chart onto your cluster (see the
+   [changelog](../changelog/index.md) for a list of Coder versions)
 
    ```bash
-   kubectl create namespace coder
-   helm install --namespace coder coder coder/coder --version <VERSION-NUMBER>
+   helm install --namespace coder coder coder/coder
    ```
 
-3. **For Production deployments:** Add the following to your helm chart so that
+3. Get a copy of your helm chart so that you can modify it; you'll need to
+   modify the helm chart to update your PostgreSQL databases (step 4) and enable
+   Dev URLs (step 5):
+
+   a. Get a copy of your existing helm chart and save as `values.yaml`: `helm
+   show values coder/coder > values.yaml`
+
+   b. Edit the `values.yaml` file as needed.
```

Comment:
We should indicate that people should remove the lines from `values.yaml` that they are not modifying. Otherwise, I'm pretty sure an upgrade may fail due to an unnecessary override. For example, some lines contain the Coder version number. We don't want to override that when updating.
```yaml
# values.yaml:146-150

envproxy:
  # envproxy.replicas -- The number of replicas to run of the envproxy.
  replicas: 1
  # envproxy.image -- Injected during releases.
  image: docker.io/coderenvs/coder-service:1.15.2
```

Reply:
To clarify, I believe _this file_ would override the values from Coder's helm chart. So, if the values.yaml has `1.15.2` and the default chart is trying to upgrade to `1.16.0`, it will still pull in the `1.15.2` image.

Reply:
@bpmct Do you think this is clearer? `Be sure to remove the lines that you are *not* modifying, otherwise
the contents of `values.yaml` will override those in the default chart.`

Reply:
Yes totally, thanks 🙏